### PR TITLE
Knollfear/bcda 740 Static Site

### PR DESCRIPTION
**Pull request by knollfear**
### Fixes [BCDA-740](https://jira.cms.gov/browse/BCDA-740)

BCDA needs a static one pager of a site.  It will ideally be flexible as needs for additional pages grow so it should be built using jekyll or a similar site builder

### Proposed changes:

Added the static site.  Too many proposed changes to list

### Change Details

Added jekyll site
added build of site to docker containers
added route for site

### Security Implications

no PII is affected, this just displays a static page.

### Acceptance Validation

Run app locally, look at site, click links, etc.  
Run axe Dequeue 508 compliance checker

See bcda-740 for Jenn's mockup
image

### Feedback Requested

Any

### CODE REVIEW

**Comment by dhgreene** [comment]
Beautiful and descriptive.  Great work!

- Speaking as one who loves Ruby, can we keep the Jekyll dependencies from loading every time we build the container?  
- The links don't render as links on any of my browsers.
- Interesting property of the VoiceOver utility: it reads a "D" all by itself as a "500."  Thus "Medicare Part B, and Part 500 claims data . . ."

File: bcda-site-static/index.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+layout: home
+title:  "Beneficiary Claims API"
```
Comment (embh) [inline bcda-site-static/index.md]: "Beneficiary Claims Data API"?

**Comment by knollfear** [comment]
> Beautiful and descriptive. Great work!
> 
>     * Speaking as one who loves Ruby, can we keep the Jekyll dependencies from loading every time we build the container?
> 
>     * The links don't render as links on any of my browsers.
> 
>     * Interesting property of the VoiceOver utility: it reads a "D" all by itself as a "500."  Thus "Medicare Part B, and Part 500 claims data . . ."

I had some malformed markdown.  I have fixed that.  I also removed the build of the site from the process and just checked in the site.  Now if you want to change the site, you need to build and commit that.  It will just be a matter of running a local docker container.

File: bcda-site-static/index.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+---
+layout: home
+title:  "Beneficiary Claims Data API"
+date:   2017-10-30 09:21:12 -0500
+description: Shared Savings Program ACOs can use this FHIR-based API to retrieve bulk claims data related to their assigned beneficiaries. Under construction; feedback invited.
```
Comment (dhgreene) [inline bcda-site-static/index.md]: Just double-checking we want the construction message here

Comment (knollfear) [reply bcda-site-static/index.md]: Elissa confirmed these are the approved words.

File: bcda/_site/index.html [hunk 3]
```diff
@@ -0,0 +1,340 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+	<!-- Metadata for Twitter -->
+	<meta name="twitter:title" content="Beneficiary Claims API" />
```
Comment (embh) [inline bcda/_site/index.html]:
```suggestion
	<meta name="twitter:title" content="Beneficiary Claims Data API" />
```

File: bcda/_site/index.html [hunk 4]
```diff
@@ -0,0 +1,340 @@
...
+	<meta name="twitter:description" content="Write an awesome description for your new site here. You can edit this line in _config.yml. It will appear in your document head meta (for Google search resu..." />
+	<meta name="twitter:image" content="http://localhost:4000/assets/img/default-share-image.jpg" />
+	<meta name="twitter:site" content="/" />
+	<meta name="twitter:card" content="summary_large_image">
+	<!-- Facebook / OpenGraph Metadata -->
+  <meta property="og:title" content="Beneficiary Claims API" />
```
Comment (embh) [inline bcda/_site/index.html]:
```suggestion
  <meta property="og:title" content="Beneficiary Claims Data API" />
```

File: bcda/_site/index.html [hunk 5]
```diff
@@ -0,0 +1,340 @@
...
+	<meta property="og:description" content="Write an awesome description for your new site here. You can edit this line in _config.yml. It will appear in your document head meta (for Google search resu..." />
+	<meta property="og:image" content="http://localhost:4000/assets/img/default-share-image.jpg" />
+	<meta property="og:url" content="/" />
+
+  <title>Beneficiary Claims API</title>
```
Comment (embh) [inline bcda/_site/index.html]:
```suggestion
  <title>Beneficiary Claims Data API</title>
```

File: bcda/_site/index.html [hunk 6]
```diff
@@ -0,0 +1,340 @@
...
+    <div class="ds-base">
+  <header class="blueberry-lime-gradient-background main-header ds-u-margin-bottom--2 ds-u-lg-margin-bottom--5">
+    <div class="api-pattern">
+      <div class="solid-container ds-u-padding-y--2 ds-u-lg-padding-y--5">
+        <div class="ds-l-container">
+          <div class="ds-l-row">
+            <div class="ds-u-color--white ds-l-sm-cold--12 ds-l-xl-col--10 ds-u-margin-left--auto ds-u-padding-x--4">
+
+
+                <h1 class="ds-display">Beneficiary Claims API</h1>
```
Comment (embh) [inline bcda/_site/index.html]:
```suggestion
                <h1 class="ds-display">Beneficiary Claims Data API</h1>
```

File: bcda/_site/index.html [hunk 7]
```diff
@@ -0,0 +1,340 @@
...
+  <!-- Desktop Navigation -->
+  <div class="ds-c-desktop-nav">
+    <!-- Logo Link Container -->
+    <a class="logo-link-container" href="/">
+      <!-- Arrow Icon
+      <svg fill="none" height="35" viewBox="0 0 35 35" width="35" xmlns="http://www.w3.org/2000/svg"><circle cx="17.5" cy="17.5" r="17" stroke="#323a45"/><circle cx="17.5" cy="17.5" fill="#323a45" r="15.1667"/><path d="m11.5 18.5h-1.5v6h14.5v-6h-1.5v4h-11.5z" fill="#f7f7f7"/><path d="m17.5 19-5-5h3.5v-7h3.5v7h3z" fill="#f7f7f7"/><g stroke="#f7f7f7"><path d="m11.5 18.5h-1.5v6h14.5v-6h-1.5v4h-11.5z"/><path d="m17.5 19-5-5h3.5v-7h3.5v7h3z"/></g></svg>
+      -->
+      <!-- Logo Text -->
+      <div class="site-title">
+        Beneficiary Care Data API
```
Comment (embh) [inline bcda/_site/index.html]:
```suggestion
        Beneficiary Claims Data API
```

File: bcda/_site/index.html [hunk 8]
```diff
@@ -0,0 +1,340 @@
...
+
+  <!-- Mobile Navigation -->
+  <div class="ds-c-mobile-nav">
+    <!-- Logo Link Container -->
+    <a class="logo-link-container" href="/">
+      <!-- Arrow Icon -->
+      <svg fill="none" height="35" viewBox="0 0 35 35" width="35" xmlns="http://www.w3.org/2000/svg"><circle cx="17.5" cy="17.5" r="17" stroke="#323a45"/><circle cx="17.5" cy="17.5" fill="#323a45" r="15.1667"/><path d="m11.5 18.5h-1.5v6h14.5v-6h-1.5v4h-11.5z" fill="#f7f7f7"/><path d="m17.5 19-5-5h3.5v-7h3.5v7h3z" fill="#f7f7f7"/><g stroke="#f7f7f7"><path d="m11.5 18.5h-1.5v6h14.5v-6h-1.5v4h-11.5z"/><path d="m17.5 19-5-5h3.5v-7h3.5v7h3z"/></g></svg>
+      <!-- Logo Text -->
+      <div class="site-title">
+        Beneficiary Care Data API
```
Comment (embh) [inline bcda/_site/index.html]:
```suggestion
        Beneficiary Claims Data API
```

File: bcda/_site/404.html [hunk 9]
```diff
@@ -0,0 +1,219 @@
...
+  <!-- Desktop Navigation -->
+  <div class="ds-c-desktop-nav">
+    <!-- Logo Link Container -->
+    <a class="logo-link-container" href="/">
+      <!-- Arrow Icon
+      <svg fill="none" height="35" viewBox="0 0 35 35" width="35" xmlns="http://www.w3.org/2000/svg"><circle cx="17.5" cy="17.5" r="17" stroke="#323a45"/><circle cx="17.5" cy="17.5" fill="#323a45" r="15.1667"/><path d="m11.5 18.5h-1.5v6h14.5v-6h-1.5v4h-11.5z" fill="#f7f7f7"/><path d="m17.5 19-5-5h3.5v-7h3.5v7h3z" fill="#f7f7f7"/><g stroke="#f7f7f7"><path d="m11.5 18.5h-1.5v6h14.5v-6h-1.5v4h-11.5z"/><path d="m17.5 19-5-5h3.5v-7h3.5v7h3z"/></g></svg>
+      -->
+      <!-- Logo Text -->
+      <div class="site-title">
+        Beneficiary Care Data API
```
Comment (embh) [inline bcda/_site/404.html]:
```suggestion
        Beneficiary Claims Data API
```

File: bcda/_site/404.html [hunk 10]
```diff
@@ -0,0 +1,219 @@
...
+
+  <!-- Mobile Navigation -->
+  <div class="ds-c-mobile-nav">
+    <!-- Logo Link Container -->
+    <a class="logo-link-container" href="/">
+      <!-- Arrow Icon -->
+      <svg fill="none" height="35" viewBox="0 0 35 35" width="35" xmlns="http://www.w3.org/2000/svg"><circle cx="17.5" cy="17.5" r="17" stroke="#323a45"/><circle cx="17.5" cy="17.5" fill="#323a45" r="15.1667"/><path d="m11.5 18.5h-1.5v6h14.5v-6h-1.5v4h-11.5z" fill="#f7f7f7"/><path d="m17.5 19-5-5h3.5v-7h3.5v7h3z" fill="#f7f7f7"/><g stroke="#f7f7f7"><path d="m11.5 18.5h-1.5v6h14.5v-6h-1.5v4h-11.5z"/><path d="m17.5 19-5-5h3.5v-7h3.5v7h3z"/></g></svg>
+      <!-- Logo Text -->
+      <div class="site-title">
+        Beneficiary Care Data API
```
Comment (embh) [inline bcda/_site/404.html]:
```suggestion
        Beneficiary Claims Data API
```

File: bcda/_site/index.html (hunk 3 above)
Comment (knollfear) [reply bcda/_site/index.html]: These should all be fixed.  In general we should avoid making changes in _site as they are built from the files in bcda-site-static.  It was fixed in the config.yml, but I failed to rebuild before committing.

**Pull request merged**